manifests/07_deployment.yaml

```diff
@@ -24,9 +24,18 @@ spec:
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
     spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
```

Comment:
is this user `nobody` ?

Reply:
yes 👍

Reply:
@s-urbaniak How do we handle NET_ADMIN pods?

```
apiVersion: v1
kind: Pod
metadata:
  generateName: network-pod-
  labels:
    name: network-pod
spec:
  securityContext:
    runAsNonRoot: true
    runAsUser: 65534
    seccompProfile:
      type: RuntimeDefault
  containers:
  - name: network-pod
    image: quay.io/openshifttest/hello-sdn@sha256:2af5b5ec480f05fda7e9b278023ba04724a3dd53a296afcd8c13f220dec52197
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        add: ["NET_ADMIN"]
  hostNetwork: true
  nodeName: node-name
```
Seems like this will throw error

Reply:
*error= warning